.changeset/chilly-dryers-begin.md

```diff
@@ -0,0 +1,28 @@
+---
+"@opennextjs/cloudflare": patch
```

Comment:
what about cutting a minor?

It would signal users that there are changes:

- calling the`initOpenNextCloudflareForDev` function,
- `getCloudflareContext` becoming sync

If you do that, please also prepare:
- a PR to update the c3 template to 0.4.x
- a PR to the doc site (copy the current content to 0.3, add the changes in the current content, add a migration from 0.3)

Reply:
done 👍